www/js/ui.js

```diff
@@ -618,7 +618,7 @@ $("#shuffleplaylist").click(function() {
 /* load channel */
 
 var loc = document.location+"";
-var m = loc.match(/\/r\/([a-zA-Z0-9-_]+)/);
+var m = loc.match(/\/\w+\/([a-zA-Z0-9-_]+)/);
```

Comment:
This is pretty disgusting code to begin with tbh.

1.) Why isn't this wrapped in an IIFE
2.) Whats wrong with `document.location.pathname.split('/')[2]`